### ISSUE TITLE
[ipfs/go-ipfs-files] Fix encoding of filename in Content-Disposition

### ISSUE THREAD

**Issue by Stebalien** [ipfs/boxo#29]
We should use [RFC5987](https://greenbytes.de/tech/webdav/rfc5987.html) to encode the filename in the Content-Disposition header. GO will already gracefully handle line-breaks but we should still be escaping better.

**Comment by kstuart** [comment]
@Stebalien do you mean in multiFileReader? The filename is encoded using url.QueryEscape so should be encoded according to standards.

Test added in this commit: 66d91bf01d8e45a77aa42da747551a467ad8d736 (#34)

Does this ticket remain an issue?

**Comment by Stebalien** [comment]
I can't remember at this point. If that's where we set Content-Disposition, then yes.

**Comment by kstuart** [comment]
Yes, that's the only place it is set, since all values are escaped using url.QueryEscape this ticket is probably stale and could be closed.
